src/Settings.test.js

```diff
@@ -1,34 +1,91 @@
 import { DeviceApi } from '../packages/device-api/index.js'
 import { createGlobalConfig } from './config.js'
 import { Settings } from './Settings.js'
-import {
-    GetAvailableInputTypesCall,
-    GetRuntimeConfigurationCall
-} from './deviceApiCalls/__generated__/deviceApiCalls.js'
+import {GetRuntimeConfigurationCall} from './deviceApiCalls/__generated__/deviceApiCalls.js'
+
+const createAvailableInputTypes = (overrides) => {
+    const base = Settings.defaults.availableInputTypes
+    return {
+        ...base,
+        ...overrides
+    }
+}
+
+const createGlobalConfigWithDefaults = (defaults) => {
+    return {
+        ...createGlobalConfig(),
+        ...defaults
+    }
+}
 
 describe('Settings', () => {
```

Comment:
interested on your opinion on the usefulness of these types of unit test? I know I added this set originally, but given how much we do integration tests now I'm curious/skeptical/cautious about adding lots of tightly coupled unit tests.

These are great additions, especially since you've improved/changed Settings, but just as a general point going forward we should discuss :)

Reply:
I think these are nice to have here. Unit tests are much faster to run on my Intel machine and I think they're also easier to write for this sort of things. We can revisit at any time if needed.